client/src/components/CommonComponents/NavBar/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { Fragment } from 'react';
+import { AppBar, Toolbar, IconButton, Grid, Box } from '@material-ui/core';
+import MoreIcon from '@material-ui/icons/MoreVert';
+import useStyles from './style';
+import Tabs from '../Tabs';
+
+export default function NavBar(props) {
+	const classes = useStyles();
+	const pathname = props.history.location.pathname;
+	const displayTab = pathname === '/' ? classes.hidden : '';
```

Comment:
I don't think its a good way, I want to ask you @mantagen if this is a good way if we decide the content based on path, not the status of logged in!

Reply:
I will try to find a way but when we merge the home page cause on that PR I have decided to call the navbar on all the pages so its kinds tricky

Reply:
Hmm

Reply:
Yes, really we want it to check if the logged in status.
Can we do that easily? Where are we storing the logged-in status?

Reply:
its gonna be for an authorization step if you want me to merge those steps together with this branch I will do it, well let's just merge this branch and then I will find a way cause I have changed the game on home page PR

Reply:
I don't think it will be a lot of changes! so if you agreed with me you can start doing it!

Reply:
its gonna rechange everything I didn't plan for authentication at this step it gonna be at another PR, this PR for Navbar only I am not gonna change anything related to sign in

Reply:
Okay we can hook it up with another PR